internal/command/config/save.go

```diff
@@ -65,6 +74,12 @@ func runSave(ctx context.Context) error {
 		return err
 	}
 
+	if flag.GetBool(ctx, "json") {
+		configfilename = strings.TrimSuffix(configfilename, ".toml") + ".json"
+	} else if flag.GetBool(ctx, "yaml") {
+		configfilename = strings.TrimSuffix(configfilename, ".toml") + ".yaml"
```

Comment:
is there a chance that `--config path/to/fly.yaml` generates `path/to/lfy.yaml.yaml` ?